src/statsbomb_snowflake/loader.py

```
@@ -0,0 +1,361 @@
+            for i in range(0, len(competitions), batch_size):
+                batch = competitions[i : i + batch_size]
+                values = []
+                for comp in batch:
+                    comp_json = json.dumps(comp, ensure_ascii=False)
+                    values.append((comp["competition_id"], comp["season_id"], comp_json, loaded_at))
+
+                cur.executemany(
+                    f"""
+                    INSERT INTO {table} (competition_id, season_id, raw_data, _loaded_at)
+                    VALUES (%s, %s, PARSE_JSON(%s), %s)
+                    """,
```

Ruff (0.14.14)

Possible SQL injection vector through string-based query construction

(S608)